api/polling/README.md

```diff
@@ -106,22 +107,20 @@ When working with order endpoints:
 
 When working with shipment endpoint:
 
-- Use `data.lines.meta.lastUpdatedAt` to identify changed shipment lines
+- Use `data.lines.meta.lastUpdatedAt` to identify changed shipment lines since the `lastUpdatedAfter` time you provided.
 - Filter by `data.status` to focus on relevant shipment statuses
 
 ### Step 3: Persist the timestamp
 
 Store the `lastUpdatedAt` value to use as `lastUpdatedAfter` in subsequent requests.
 
+When the poll response is empty, keep using the same timestamp until you get results.
+
 #### Implementation tips
 
 - The timestamp has format `YYYY-MM-DDThh:mm:ss.SSSZ` (ISO 8601)
 - Store this in **persistent storage** (database, file system, etc.)
 - Your storage solution must survive application restarts or crashes
 - If the response contains no data, reuse your current `lastUpdatedAfter` value
 - For first-time polling, use a historical date to fetch existing records
-
-#### Edge cases
-
-- **Initial setup**: Set `lastUpdatedAfter` to retrieve historical data as needed
 - **Empty responses**: Keep using the same timestamp until you get results
```

Comment:
@marcmatt maybe a nitpick, but isn't this the same as 
`- If the response contains no data, reuse your current `lastUpdatedAfter` value` ?

Reply:
removed, I missed it